classifier/mixing_in_negative_samples.py

```diff
@@ -0,0 +1,133 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import argparse
+import csv
+import os
+import random
+
+import numpy as np
+
+
+""" Mix synthetic negative samples into training data."""
+
+
+def combine_two_tsv_files(original_file_path, append_file_path, number_of_new_lines,
+                         output_file_path):
+    """Mix some of the lines in a file to another file, and write to a target path.
+    Args:
+      original_file_path: the file where all the lines will be present in the 
+        target file
+      append_file_path: the file where some of the lines will be present in the target 
+        file
+      number_of_new_lines: the number of lines from the second file that will be 
+        present in the target file
+      output_file_path: the path where the target file will be saved
+    """
+    with open(original_file_path) as f:
+        original_lines = f.readlines()
+    with open(append_file_path) as f:
+        new_lines = f.readlines()
+    
+    if number_of_new_lines < 0:
+        raise ValueError("The target ratio is too small.")
+    if number_of_new_lines > len(new_lines):
+        raise ValueError("The target ratio is too big given the number of synthetic data.")
+    
+    random.shuffle(new_lines)
+    shuffled_index = list(range(len(original_lines) + number_of_new_lines))
+    random.shuffle(shuffled_index)
```

Comment:
can you comment what you are doing here? it's not very obvious.

Reply:
I am reshuffling the dataset after adding new samples (see " for index in shuffled_index: " on line 57)